src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -96,17 +96,17 @@ def _get_all_property_names_and_resource_ids_one_resource(
     return prop_dict
 
 
-def _check_if_all_class_types_exist(
+def _check_all_class_types_exist(
```

Comment:
Again, the sentence "Check all class types exist" is grammatically not understandable. Perhaps you mean: "Check all class types"?